src/ACConfig.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace secondtry
+{
+    public class ACConfig
+    {
+        public string userpath { get; set; }
+        public string Costumer { get; set; }
```

Comment:
Ich glaube das war mein fehler :) sorry. war spät. ^^


```suggestion
         public string Customer{ get; set; }
```